README.md

```diff
@@ -1,35 +1,83 @@
-okta-spring-security
-====================
+Okta Spring Boot Starter
+========================
 
-Still some work to be done on this README.
+Okta's Spring Boot Starter will enable your Spring Boot application to work with an Okta access token via an OAuth implicit flow.  Jump to our [quickstart](https://developer.okta.com/quickstart/#/angular/java/spring) to see how to configure various clients or follow along below and to use curl.
```

Comment:
Is the link to the Quickstart correct? The landing page only seems to show dependencies and properties for this starter.

- "follow along below and" should be "follow along below".